static/js/input_pill.js

```diff
@@ -96,23 +97,30 @@ export function create(opts) {
 
             store.pills.push(payload);
 
-            const has_image = item.img_src !== undefined;
+            const has_image = payload.item.img_src !== undefined;
 
             const opts = {
                 id: payload.id,
-                display_value: item.display_value,
+                display_value: payload.item.display_value,
+                deactivated_user_pill: false,
                 has_image,
             };
 
             if (has_image) {
-                opts.img_src = item.img_src;
+                opts.img_src = payload.item.img_src;
```

Comment:
Why does this code switch from referencing `item.foo` to `payload.item.foo`?  If we need to do that, it might be worth doing in its own commit for readability.

Reply:
This was actually done by the old PR. While rebasing didn't noticed this. I will undo this.